### PR TITLE
Update holster, Golang to 1.22.

### PR DESCRIPTION
**Pull request by Baliedge**
Bring Kafka-Pixy out of the early days of Golang.

Update Holster and update Golang to latest.

Depends on:
- https://github.com/mailgun/holster/pull/194

### CODE REVIEW

File: .github/workflows/build_and_push.yml [hunk 1]
```diff
@@ -14,41 +14,41 @@ jobs:
...
           # the containers started by docker compose on localhost
           driver-opts: network=host
       # Login to the registry
       - name: Login to GitHub Container Registry
-        uses: docker/login-action@v2
+        uses: docker/login-action@v3
         with:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.PKG_WRITER_PAT }}
```
Comment (vtopc) [inline .github/workflows/build_and_push.yml:32]:
```suggestion
          password: ${{ secrets.CI_PAT }}
```

Comment (Baliedge) [reply .github/workflows/build_and_push.yml:32]: Unfortunately, that didn't fix the Docker login error: 
image

**Comment by Baliedge** [comment]
Build is breaking due to `docker/login` action failing on bad credentials in GitHub secrets.  Seeing as this repo hasn't been updated in over 2 years, it may be best to leave this update for a later time when it is required.

**Comment by Baliedge** [comment]
Thanks @jtet for fixing the docker secret.  Build is passing now.

**Comment by Baliedge** [comment]
Closing this PR for now until the Holster/Golang update becomes a requirement.  Good to know the build is fixed.

**Pull request closed without merging**